homeassistant/components/netatmo.py

```diff
@@ -1,5 +1,5 @@
 """
-Support for the Netatmo devices (Weather Station and Welcome camera).
+Support for the Netatmo devices (Weather Station, Welcome camera and Thermostat).
```

Comment:
I think we can remove what there is between the brackets as there will be more and more device supported